### DISCUSSION TITLE
Great execution!

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#3] [General]
Just wanted to drop a quick "thank you" for this one! There's been a few plugins that tried to solve this in the past, but this is super elegant, polished, and works great on both the front and back end. Really appreciated!

**Comment by aaronbushnell** [comment]
That is really kind of you, @kylecotter! Thanks so much for the love! We hope it's useful to you and can't wait for it to get better. 🥰
